### ISSUE TITLE
.hpp files not visible in workspace tree

### ISSUE THREAD

**Issue by nem0** [slynch8/10x#43]
We have a lot of .hpp files in our VS solution. When I open the solution in 10x, the files are not visible in workspace tree. `Find File` finds only single .hpp file, while we have 10s of them in solution. `C++ Goto Source/Header` switches correctly from .cpp to .hpp

P.S: 10x looks awesome, it misses a lot of stuff VS has, but it's definitely going in good direction. I wich VS was this fast. Great work.

**Comment by slynch8** [comment]
Hi, thanks for the feedback. Still a way to go, but it's getting there.

.hpp files are working in my tests. Can you check how it appears in the vcxproj file? Should be something like <ClInclude Include="MyFile.h" />

**Comment by nem0** [comment]
```
    <ClInclude Include="pkg\pkg_entity_types.h" />
    <hpp Include="pkg\pkg_animation_manager.hpp" />
    <hpp Include="pkg\pkg_geom_manager.h" />
    <ClInclude Include="pkg\pkg_hit_mask.h" />
```
Now that I see it, those files have custom `Item type` in VS

**Comment by slynch8** [comment]
fixed (0.22.26) it will now accept elements of any name if it looks like the Include attribute is a c++ file.
